### PR TITLE
Merge rel 1.0.0 into feature/msbuild

### PR DESCRIPTION
**Pull request by livarcocc**
cc @eerhardt @piotrpMSFT @brthor

### CODE REVIEW

**Comment by brthor** [comment]
The deb tool changes made to rel/1.0.0 will break packaging in feature/msbuild without appropriate changes

**Comment by livarcocc** [comment]
Let's look through it together. I removed the DebTargets.cs file already locally. But I don't know if there anything else that needs to happen.

**Comment by brthor** [comment]
There was significant change to how the deb packages are generated, an msbuild task needs to replace the .sh files that are being called by Deb.targets

**Comment by eerhardt** [comment]
@livarcocc - you can just cherry-pick b7075b8 into feature/msbuild... to get you unblocked.

Although, it would be good to get rel/1.0.0 merged into feature/msbuild.

File: src/dotnet/commands/dotnet-new/CSharp_Console/Program.cs [hunk 1]
```diff
@@ -1,12 +1,9 @@
 ﻿using System;
 
-namespace ConsoleApplication
+class Program
```
Comment (eerhardt) [inline src/dotnet/commands/dotnet-new/CSharp_Console/Program.cs]: This change should be ported into https://github.com/dotnet/cli/tree/feature/msbuild/src/dotnet/commands/dotnet-new/CSharp_MSBuild as well.

**Comment by brthor** [comment]
Talked offline. I'll make this deb change later in the week, I won't have time today.

**Comment by livarcocc** [comment]
I will cherry pick it. talked to Bryan offline.

**Pull request closed without merging**